src/LagoonCommands.php

```diff
@@ -67,11 +77,10 @@ public function __construct() {
     // Get default config.
     $lagoonyml = $this->getLagoonYml();
     $this->api = $lagoonyml['api'] ?? 'https://api.lagoon.amazeeio.cloud/graphql';
```

Comment:
to accommodate https://github.com/uselagoon/build-deploy-tool/pull/193

```suggestion
    $this->api = getenv('LAGOON_CONFIG_API_HOST') ?? $lagoonyml['api'] ?? 'https://api.lagoon.amazeeio.cloud/graphql';
```